jaeger/info.go

```diff
@@ -0,0 +1,48 @@
+package jaeger
+
+import (
+	"errors"
+	"net/http"
+	"net/url"
+	"path"
+
+	"github.com/kiali/kiali/config"
+)
+
+func getJaegerInfo(client http.Client, endpoint *url.URL) (*JaegerInfo, int, error) {
```

Comment:
can we elaborate a little bit with the changes in our dependencies to Jaeger? I'm not sure to have followed everything:

- do we still need a configured external URL or not? In other words, are we going to keep some links to jaeger from within the browser?

- What does mean "Integration" here: is it that any kind of integration with Jaeger is enabled/disabled when this flag is true/false? For instance in the Spans overlay that I did, is it supposed to be turned off when Integration is false? Even when the in_cluster_url is working correctly but not the external one?

Reply:
In that way,

If user configure the InClusterUrl, the user will see the traces and the stuff provided by Kiali,
If user configure the external URL the user will see the links to the Jaeger UI in kiali.

Casuistics

- User can set only the internal URL => The user will see the traces tab
- User can set only the  external URL => The user will see the traces tab but this tab will be a link to the JaegerUI traces for that service, and he'll see the distributed tracing link in the menu
- User can set boths

For example /api/jaeger returns

```json
{
"url":"http://localhost:16686/",
"namespace_selector":true,
"integration":false,
"integration_message":"Error with internal connection with Jaeger: Get http://localhost:16686/api/services: dial tcp [::1]:16686: connect: connection refused"}
```

This means that user will see links to Jaeger UI in Kiali.
The integration is false, this means that kiali can't reach the jaeger service internally and the integration_message throw the cause, so the user won't see the traces tab with the scatter plot, this tab will be a link to the Jaeger UI
And the namespace_selector indicate that we need to look the traces by service.namespace

cc @lucasponce @jotak

Reply:
thanks!

Reply:
@aljesusg you wrote ` If user configure the external URL the user will see the links to the Jaeger UI in kiali.` but it doesn't seem to be true today: when no external URL is provided, I don't see the Traces tab at all (whereas I can see the spans within metrics charts). Should I open a ticket?

Reply:
Yes please open a ticket, if you have the internal configured but not the external one you should see the traces tab.